StyleCop.Analyzers/StyleCop.Analyzers.Test.CSharp12/SpacingRules/SA1008CSharp12UnitTests.cs

```diff
@@ -3,9 +3,26 @@
 
 namespace StyleCop.Analyzers.Test.CSharp12.SpacingRules
 {
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.CodeAnalysis.Testing;
     using StyleCop.Analyzers.Test.CSharp11.SpacingRules;
+    using Xunit;
+
+    using static StyleCop.Analyzers.Test.Verifiers.StyleCopCodeFixVerifier<
+        StyleCop.Analyzers.SpacingRules.SA1008OpeningParenthesisMustBeSpacedCorrectly,
+        StyleCop.Analyzers.SpacingRules.TokenSpacingCodeFixProvider>;
 
     public partial class SA1008CSharp12UnitTests : SA1008CSharp11UnitTests
     {
+        [Fact]
+        [WorkItem(3743, "https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3743")]
+        public async Task TestTupleUsingAliasAsync()
+        {
+            const string testCode = @"
+using TestAlias = (string X, bool Y);";
```

Comment:
💡 This should be tested similar to others where it also tests that a diagnostic is reported if there is no space, and the code fix inserts the missing space.

Reply:
Updated